packages/ember-metal/lib/property_events.js

```diff
@@ -108,6 +111,13 @@ function propertyDidChange(obj, keyName) {
   }
 
   markObjectAsDirty(m);
+
+  if (isEnabled('ember-glimmer-detect-two-way-reflush') ||
+      isEnabled('ember-glimmer-allow-two-way-reflush')) {
+    if (has('glimmer-runtime')) {
```

Comment:
I think the solution is either

1. in emberjs-build, make sure we hard code these to false unless glimmer is enabled, or
2. in application boot, if these flags is enabled, do an assert for `has('glimmer-runtime')`

Then the functions can be no-ops